src/Domain/CampaignService.php

```diff
@@ -60,6 +60,13 @@ public function renderCampaign(CampaignDomainModel $campaign): array
         $sfCampaignData = $campaign->getSalesforceData();
         $sfCharityData = $sfCampaignData['charity'];
 
+        try {
+            $websiteURi = $charity->getWebsiteUri()?->__toString();
```

Comment:
```suggestion
            $websiteUri = $charity->getWebsiteUri()?->__toString();
```
Slightly less confusing case, not a blocker.